### DISCUSSION TITLE
Drag-and-drop tasks in Markdown task lists not working

### DISCUSSION THREAD

**Discussion by FSDKO** [community/community#44569] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Body

This feature no longer works:
<https://github.blog/2016-05-27-drag-and-drop-tasks-in-markdown-task-lists/>

> You can now move checklist items around just by dragging and dropping them. Reorder items quickly and easily without editing the original comment's Markdown. How to re-order task list items…

I testet it in an public and in a private "for free" repository. 

There is no longer the area to grab the task and it ist not possible to move it:
![image](https://user-images.githubusercontent.com/26324364/212880537-386ef1f5-4404-4fcb-8953-7c51aa729f55.png)

**Comment by flatheadmill** [comment]
Can confirm. It has made it very hard to continue to use our GitHub Projects boards. Been this way for a week now. I'm glad someone noticed.

**Comment by SitronX** [comment]
I hope it is a bug and it gets fixed soon. It is a really annoying to move tasks manually by copy-pasting the raw text

**Comment by flatheadmill** [comment]
Actually very difficult when you're using tracking issues so that the Markdown check list entries look like `- [ ] #453`.

**Comment by guylepage3** [comment]
+1. Hoping this gets resolved soon.

I sent in a support ticket for this a couple of weeks ago. https://support.github.com/ticket/personal/0/1963834

**Reply by chalin** [reply]
Thanks for submitting the ticket. I tried viewing it but I can't, so it's probably not publicly viewable.
Any responses from GH?

**Comment by cyrilsjones** [comment]
Critical in the world of ever changing priorities and managing expectations.

**Comment by flatheadmill** [comment]
🚨Look for this icon...🚨

<img width="103" alt="Screen Shot 2023-02-02 at 2 22 09 PM" src="https://user-images.githubusercontent.com/5694411/216441348-11b0de67-b4bc-4d7c-8d45-ac47726d2d5f.png">

If you have that in your edit toolbar, you have the new beta task lists which require a new special markdown.

<img width="553" alt="Screen Shot 2023-02-02 at 2 23 35 PM" src="https://user-images.githubusercontent.com/5694411/216441613-37fa628e-1e27-45dd-9d89-ed053afeb77e.png">

Someone in your organization, possibly you, signed up for the alpha/beta for the new tasks list and you got it. Pity the old task lists don't still DnD, but I'm happy to beta test the new ones and I'm converting my active Issues to the new format.

**Reply by chalin** [reply]
Thanks! Right, no DnD, _and_ it flattens all my nested list items. :-/ Maybe they'll improve over time. :)

**Reply by guylepage3** [reply]
Lol. That icon does not exist on issue comment boxes. Lol

**Reply by chalin** [reply]
FWIW, it worked for me even without the icon.

**Reply by gomo** [reply]
The icon does not exist on my issue too, and it did not work for me.

**Reply by FSDKO** [reply]
The new tasklist is in private beta at the moment:
https://docs.github.com/en/issues/tracking-your-work-with-issues/about-tasklists

I hope they release it very soon.

... maybe they kicked the old tasklist because it made conflicts with the new one.

**Reply by flatheadmill** [reply]
I [commented](https://github.com/orgs/community/discussions/39106#discussioncomment-4836349) in the tasklists beta discussion last week. That discussion would be the one to follow to see how tasklists redux sorts itself out.

**Reply by gomo** [reply]
I have registered for the wait list for Tasklists from the following page.

https://docs.github.com/en/issues/tracking-your-work-with-issues/about-tasklists

A few days later I received an email "Welcome to the Tasklist Private Beta" and I was able to use new taskists. 

````
```[tasklist]
### Tasks
- [ ] Stakeholders identified
- [ ] Funding identified
- [ ] Owning team identified
```
````
This can be reordered.

The old Tasklist is still not reordable.

**Comment by azenMatt** [comment, marked as answer]
Hi all - we have eyes on this and are working on a resolution.

**Reply by FSDKO** [reply]
Hey it works again. Thank you very much.
🎉

**Reply by raintreesea** [reply]
It's not working for me in the cedarai repo. When I mouse over a task that has not yet been turned into an issue, I don't see a reorder icon or a create-issue icon.  I also don't see the number of tasks associated with an issue at the top.

It appears that we are not on the Tasklist beta, but I tried that anyway just in case, and no luck :/

**Reply by raintreesea** [reply]
The tasklists are working today! Thank you!
![image](https://user-images.githubusercontent.com/3772234/235494156-7d711183-b8c7-48a5-bf8a-5c10218e7544.png)
![image](https://user-images.githubusercontent.com/3772234/235494212-f07fdbac-2fd2-4381-8389-045e27c61e8d.png)

**Comment by gavindidrichsen** [comment]
Hi All,

I've been using the new beta Tasklists for some time now--and love it.  However, after creating a new private organization, creating an issue, I've discovered that I can't create the tasklist in here as in my other organization.  For example,

In my new organization editing an issue I see:

![image](https://github.com/community/community/assets/16652757/de38abfa-a627-44f6-b641-1d73ad375a0c)

and then

![image](https://github.com/community/community/assets/16652757/9170119f-91fb-4b24-a308-c46b30a20d95)

However, my older private organization can use the Tasklist:

![image](https://github.com/community/community/assets/16652757/1ac50429-7c82-4a5c-8de1-7120f5f0975b)

![image](https://github.com/community/community/assets/16652757/fd7666dd-3c2b-4401-8a39-6285a44a8fe5)

Is there something I need to do with my new organization to enable the Tasklists?
